### ISSUE TITLE
New CSS organization

### ISSUE THREAD

**Issue by davidgatti** [adbloomandrew/teachparttime#20]
@Serhii-Hulko I did convert the project in to a ExpressJS app, deployed on Heroku. The URLs are the same:

- https://dev.teachpart-time.com/
- https://potato.teachpart-time.com/

From the heroku branch i did a new style branch called [heroku-style](https://github.com/adbloomandrew/teachparttime/tree/heroku-style) so you can work on your style there.

In the [CSS folder](https://github.com/adbloomandrew/teachparttime/tree/heroku-style/public/css) I made the following:

- we have the default Boostrap 4 CSS that will be used for each client and on all the pages - please do not change this.
- I made a `clients` folder where all the custom styles for each clients should be put. 

Tasks:
- [x] Please remove the boostrap style rom the dev.css file and just keep the custom css. 
- [x] Please make a small difference for the style in the potato.css file so we can test how this switching works. 

let me know if you have any questions.

/cc: @aabony

**Comment by Serhii-Hulko** [comment]
@davidgatti @davidgatti I can't turn on project. I tried to run `npm start` command and cot the message `20:57:20 web.1   |  Worker 1 is listening on port 5000` but it doesn't work on this port. I used url with localhost and my local ip address. Can you help me?

**Comment by Serhii-Hulko** [comment]
@davidgatti I have removed bootstrap styles from dev.css and modified potato.css, so you can test if it works. The other changes will be added to the same branch after you help me with project running on localhost

**Comment by davidgatti** [comment]
@Serhii-Hulko I updated the README in the [Heroku](https://github.com/adbloomandrew/teachparttime/tree/heroku) branch, now the README explains in details how to go about working with Heroku, on your local machine.

**Comment by davidgatti** [comment]
@Serhii-Hulko perfect I made a PR, and deployed on Heroku. And all looks as it should and the small change for Potato is perfect. 

So I hope that this work flow now is clear, each client will get their own customized style. 

One request that I have - there is no need to minimize the CSS, makes it hard to understand what is going on. And since we are not Google or Facebook, shaving off few bytes won't make a big difference :) 

I enabled compression for the server, so every white space will be easily compressed.
